src/Client.php

```diff
@@ -107,7 +107,7 @@ private function createDefaultDecider($maxRetries)
     private function initClient($url, $token, array $options = [])
     {
         // Initialize handlers (start with those supplied in constructor)
-        if (isset($options['handler']) && $options['handler'] instanceof HandlerStack) {
```

Comment:
Tohle bylo imho vzdycky blbe, je to vybrany z docker bundle. Nejspis se to stejne nikde nepouziva